apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -41,21 +47,32 @@ Page {
                 active: !AccessPolicy.browserEnabled
             }
 
-            TextField {
+            BrowserListItem {
                 id: homePage
-                enabled: AccessPolicy.browserEnabled
 
                 //: Label for home page text field
                 //% "Home Page"
                 label: qsTrId("settings_browser-la-home_page")
-                text: homePageConfig.value == "about:blank" ? "" : homePageConfig.value
-
-                inputMethodHints: Qt.ImhNoPredictiveText | Qt.ImhNoAutoUppercase | Qt.ImhUrlCharactersOnly
-
-                onTextChanged: homePageConfig.value = text || "about:blank"
-
-                EnterKey.iconSource: "image://theme/icon-m-enter-close"
-                EnterKey.onClicked: focus = false
+                //% "Default"
+                value: homePageConfig.value === "about:blank" ? qsTrId("sailfish_browser-la-home_page_default") : removeProtocolTypeFromUri(homePageConfig.value)
+                iconSource: "image://theme/icon-m-home"
```

Comment:
There's an animation glitch when the item is changed from a long URL, to "Default home page". In this case the little "Home" icon jumps to the floor before moving up to the centre of the item. It must be an issue with `BrowserListItem`; would it be possible to fix it?

Reply:
Most likely this is a problem in `BrowserListItem`

Reply:
@llewelld  added an overload alias: `property alias animationDuration: valueButton._duration` in` BrowserListItem`. Now "Home" icon not jumps.

Reply:
Thanks for taking a look at this. I'm still getting the icon jumping position (but in a slightly different way). It's more obvious if text size is set to Huge.

I agree with you that this is a problem in `BrowserListItem`, so if you prefer we can leave it as it is here and look into the jump separately.

Reply:
@llewelld it seems to me better to abandon the `BrowserListItem` in favor of the ComboBox. The problem is being solved.